src/js/themes/base.js

```diff
@@ -1321,6 +1321,41 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         margin: 'none',
         fill: true,
       },
+      size: {
+        small: {
+          pad: {
+            top: 'medium',
+            bottom: 'small',
+          },
+          subtitle: {
+            size: 'small',
+            margin: 'none',
```

Comment:
Is `margin: 'none'` needed? I was expecting that "small" and "large" would baseline with the defaults and only adjust specific properties that were different.

Reply:
That's fair. Let me update the logic.